### DISCUSSION TITLE
Easy Export and Import Alert Rules

### DISCUSSION THREAD

**Discussion by kuMunasinghe** [grafana/grafana#52439] [Alerting]
There is no convenient method of exporting and importing alert rules.I think this is a must feature and if it is available I am sure that everybody will appreciate it.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
